pkg/controller/vm_controller.go

```diff
@@ -569,6 +585,47 @@ func (r *VMReconciler) buildVMPod(ctx context.Context, vm *virtv1alpha1.VirtualM
 					},
 				},
 			})
+
+			if iface.Vhostuser != nil {
+				type nadConfig struct {
+					Type                      string `json:"type"`
+					VhostuserSocketVolumeName string `json:"vhost_user_socket_volume_name,omitempty"`
+					VhostuserSocketName       string `json:"vhost_user_socket_name,omitempty"`
+				}
+
+				var cfg nadConfig
+				if err := json.Unmarshal([]byte(nad.Spec.Config), &cfg); err != nil {
+					return nil, fmt.Errorf("unmarshal NAD config: %s", err)
+				}
+
+				switch cfg.Type {
+				case "kube-ovn":
+					if vmPod.Spec.NodeSelector == nil {
+						vmPod.Spec.NodeSelector = map[string]string{}
+					}
+					vmPod.Spec.NodeSelector["ovn.kubernetes.io/ovs_dp_type"] = "userspace"
+					vmPod.Annotations["ovn-dpdk.default.ovn.kubernetes.io/mac_address"] = iface.MAC
+
+					vmPod.Spec.Volumes = append(vmPod.Spec.Volumes, corev1.Volume{
+						Name: cfg.VhostuserSocketVolumeName,
+						VolumeSource: corev1.VolumeSource{
+							EmptyDir: &corev1.EmptyDirVolumeSource{},
+						},
+					})
+					volumeMount := corev1.VolumeMount{
+						Name:      cfg.VhostuserSocketVolumeName,
+						MountPath: "/var/run/vm",
```

Comment:
It's not a good mount path

Reply:
-> /var/run/vhost-user-socket ?

Reply:
/var/run/vhost is better?

Reply:
Maybe `/var/run/vhost-user`? vhost is another type of interface.

Reply:
👌